modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/type/PodcastsSortType.kt

```diff
@@ -47,6 +39,14 @@ enum class PodcastsSortType(
         folderComparator = Comparator { _, _ -> 0 },
         analyticsValue = "episode_release_date",
     ),
+    DATE_ADDED_OLDEST_TO_NEWEST(
```

Comment:
```suggestion
    DATE_ADDED_NEWEST_TO_OLDEST(
```
Should we rename this to match the new sort order?

Reply:
Good point. Renamed it in 85fae2f.